mtags/src/main/scala/scala/meta/internal/metals/MetalsVirtualFile.scala

```diff
@@ -0,0 +1,16 @@
+package scala.meta.internal.metals
+
+import scala.meta.pc.VirtualFile
+import scala.meta.inputs.Input
+
+case class MetalsVirtualFile(
```

Comment:
I believe we use the `Impl` suffix for similar instances, so this would be `VirtualFileImpl`, for consistency.